apim-operator/pkg/controller/add_security.go

```diff
@@ -0,0 +1,10 @@
+package controller
```

Comment:
don't we maintain the licence header?

Reply:
Since this file is autogenerated and not modified by us we are not adding license header